### ISSUE TITLE
Feature Request: Exclude certain folders in the workspace from linting

### ISSUE THREAD

**Issue by sriprasannaMG** [jbenden/vscode-c-cpp-flylint#33]
Hi,
First of thank you very much for the is cool extension. I set it up for clang analysis in a few minutes for a pretty big project. However I am unable to figure of how to exclude the analysis reports on certain folders (containing system headers) to avoid the report from getting polluted

**Comment by jbenden** [comment]
Hello,

The latest extension automatically discards reports from system headers, by not permitting reports to any files outside of the workspace.

Are your system headers inside a folder within your workspace? Is it possible to place them outside of the workspace, or do you require a new option for paths to explicitly ignore inside of a workspace?

Thank you for your time and patience,
-Joe

**Comment by sriprasannaMG** [comment]
Hi Joe,
Not reporting the files outside of workspace is greatly helping me. However I do have certain system header files in my project that I want to navigate to and see whats in it but not edit it (also helping the intellisense). So it would be 'nice to have' them excluded from linting. Same applies for certain parts of the source code which my I cannot take responsibility for. It does conflict to the idea of maintaining the overall good 'Project quality' and not just focus on a 'module quality' but in reality this feature might be helpful for big projects.  

Thanks again for the amazingly fast response and your commitment!

**Comment by jbenden** [comment]
Thank you for the excellent explanation. The feature should be in the next release.

**Comment by jbenden** [comment]
A new version of the extension is now available (version 0.6.0), which includes this specific change.

Please have a look at the setting:  `"c-cpp-flylint.excludeFromWorkspacePaths"`

It would be used like:

```
    "c-cpp-flylint.excludeFromWorkspacePaths": [
        "third-party"
    ],
```
